### DISCUSSION TITLE
#  min_shared_counts cutoff

### DISCUSSION THREAD

**Discussion by TarekGharsalli** [theislab/scvelo#718] [Q&A]
Hello, 
I'm new user of scvelo 
Does any one has an idea how to choose min_shared_counts when we run scv.pp.filter_and_normalize.
I tried many values 20, 5, 1 and each times I get different direction of the arrows.
My second question : It's mandatory to filter genes with low count  ?
Thanks

**Comment by WeilerP** [comment]
@TarekGharsalli, as far as I am aware, there is no recipe on how to choose the value. I would also not focus too much on the direction of arrows but rather how the phase portraits look like, *i.e.* if they include any info that can be leveraged for RNA velocity analysis.

Regarding you second question: Yes, you should filter genes by low counts and only keep highly variable ones for the following RNA velocity analysis. This is a standard step of preprocessing single-cell datasets.

**Reply by TarekGharsalli** [reply]
Thanks @WeilerP  :)
